src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs

```diff
@@ -140,6 +183,27 @@ static void ProduceLinkerAnalysisWarning()
             return testProject;
         }
 
+        private TestProject CreateTestProjectWithAotLibrary(string targetFramework, string projectName, bool isExecutable)
+        {
+            var testProject = new TestProject()
+            {
+                Name = projectName,
+                TargetFrameworks = targetFramework,
+                IsExe = isExecutable
```

Comment:
I think a ClassLibrary cannot be an executable. Therefore `IsExe` should not be `isExecutable` for `CreateTestProjectWithAotLibrary`. See https://github.com/dotnet/sdk/blob/main/src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs#L1787-L1826